habutax/forms/ty2021/f8889.py

```diff
@@ -0,0 +1,56 @@
+from habutax.form import Form
+from habutax.inputs import *
+from habutax.fields import *
+
+class Form8889(Form):
+    form_name = "8889"
+    tax_year = 2021
+
+    def __init__(self, **kwargs):
+        instance = kwargs['instance']
+        assert(instance in ['you', 'spouse'])
+        you = "you" if instance == "you" else "your spouse"
+        your = "your" if instance == "you" else "their"
+
+        inputs = [
+            BooleanInput('part_2_needed', description=f'Do {you} need to fill out part 2 of Form 8999? Part 2 is used to report HSA distributions.'),
```

Comment:
I don't think part_2_needed and part_3_needed are being used anywhere, so we're not checking to make sure they're unused.